### PR TITLE
Rename investmentFee to investmentTolerance

### PR DESCRIPTION
**Pull request by gabrielpoca**
The concept of "investment fee" is very misleading, and it has been the source of many frustrating discussions these past months. I think changing the idea to include the word "tolerance" may point the reader in a better direction.

### CODE REVIEW

**Review by 0xrin1 — APPROVED** [review]
LGTM

just merged the CustomErrors branch so will need a pull from main

File: contracts/Vault.sol [hunk 1]
```diff
@@ -112,7 +112,7 @@ contract Vault is
     uint256 public accumulatedPerfFee;
 
     /// Investment fee pct
```
Comment (naps62) [inline contracts/Vault.sol:114]: the comment needs to be updated as well

**Comment by naps62** [comment]
I think the name is better, but it's still very unclear. It's not a "tolerance on the investment", which is exactly what the name suggests. rather, it's a tolerance on how much we allow as a loss due to swap fees, etc.
With that in mind, I don't think the new name helps in any way for people coming in fresh to the codebase

**Comment by CrisBRM** [comment]
> I think the name is better, but it's still very unclear. It's not a "tolerance on the investment", which is exactly what the name suggests. rather, it's a tolerance on how much we allow as a loss due to swap fees, etc. With that in mind, I don't think the new name helps in any way for people coming in fresh to the codebase

lossTolerance seems clearer imo

**Comment by gabrielpoca** [comment]
@CrisBRM @naps62 updated, I'm sorry I had to setup prettier for ts files, to many conflicting changes

**Pull request merged**